libs/common-services/src/main/java/uk/gov/di/ipv/core/library/gpg45/Gpg45Scores.java

```diff
@@ -88,7 +86,12 @@ public int getVerification() {
     }
 
     public List<Evidence> getEvidences() {
-        return evidences;
+        return evidences.stream()
+                .sorted(
+                        Comparator.comparingInt(Gpg45Scores.Evidence::getStrength)
+                                .thenComparingInt(Gpg45Scores.Evidence::getValidity)
+                                .reversed())
+                .collect(Collectors.toList());
```

Comment:
It looks like one of the constructors already sorts the evidence - could we make all the other constructors use that one to avoid having to sort it every time you read this field?

Reply:
(it's pretty suspicious that this isn't already the case!)

Reply:
That makes sense, and we assume this remains immutable